Documentation/packaging.md

```diff
@@ -0,0 +1,65 @@
+# packaging rkt
+
+## Build-time dependencies
+
+#### Basic build-time dependencies
+
+- bash
+- go
+- gofmt
+- file
+- git
+- gpg
+
+#### Additional build-time dependencies for the coreos flavor
+
+wget, gpg, mktemp, md5sum, cpio, gzip, unsquashfs, sort
+
+#### Additional build-time dependencies for the src flavor
+
+intltoolize, libtoolize and all systemd dependencies
+
+## Run-time dependencies
+
+#### Basic run-time dependencies
+
+Linux configured with [suitable options](Documentation/hacking.md#run-time-requirements)
+
+#### Additional run-time dependencies for the host flavor
+
+- systemd >= v222 with systemctl, systemd-shutdown, systemd, systemd-journald.
+- bash
+
+## Ownership and permissions of rkt directories
+
+In general, subdirectories of `/var/lib/rkt` should be created with the same ownership and permissions as done by `rkt install`, see [directory list](https://github.com/coreos/rkt/blob/master/rkt/install.go#L44).
+
+The package should create a group `rkt` and `/var/lib/rkt` should belong to group `rkt` with the `setgid` bit set (`chmod g+s`)
+
+TODO: add better explanation
+
+## Offline builds
+
+By default, rkt compilation will download a CoreOS PXE image from internet and extract some binaries such as `systemd-nspawn` and `bash`. However, some packaging environments dont allow internet access during the build.
+
+To solve this, you can download the CoreOS PXE image before starting the build process and use the following options:
+
+```
+./configure --with-coreos-local-pxe-image-path=/data/coreos_production_pxe_image.cpio.gz --with-coreos-local-pxe-image-systemd-version=v222
+```
+
+## Code bundle
```

Comment:
Ok, I should say something about it.

Debian is trying to package each individual thing:
https://wiki.debian.org/rkt-packaging